packages/react/src/Icon/index.ts

```diff
@@ -1,2 +1,2 @@
 export { Icon } from './Icon'
-export type { IconProps } from './Icon'
+export type { IconProps, IconSize } from './Icon'
```

Comment:
We don't really need to export `IconSize` I think, we already have `IconProps['size']`